### PR TITLE
Add author information

### PR DESCRIPTION
**Pull request by lezzago**
Signed-off-by: Ashish Agrawal <ashisagr@amazon.com>

### Description
Add author details for blog posts
 
### Issues Resolved
N/A

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by nateynateynate — CHANGES_REQUESTED** [review]
Frontmatter uses `.jpeg` but uploaded photo is `.jpg` - this won't match up.

File: _authors/ashisagr.markdown [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Ashish Agrawal
+short_name: ashiasgr
+photo: '/assets/media/authors/ashisagr.jpeg'
```
Comment (nateynateynate) [inline _authors/ashisagr.markdown:4]: The uploaded file ends in `.jpg` - this will have to match. One of the two will have to change. :-)

**Review by nateynateynate — CHANGES_REQUESTED** [review]
ashisagr.jpeg -> ashisagr.jpg

File: _authors/ashisagr.markdown (hunk 1 above)
Comment (lezzago) [reply _authors/ashisagr.markdown]: @nateynateynate I fixed it. Thanks for catching it.

**Review by krisfreedain — APPROVED** [review]
looks good

**Pull request merged**